x/protorev/keeper/keeper_test.go

```diff
@@ -901,6 +901,53 @@ func (s *KeeperTestSuite) setUpPools() {
 	s.Require().NoError(err)
 }
 
+func (s *KeeperTestSuite) CreateCLPoolAndArbRouteWithManyTicks() {
```

Comment:
do you think we should also test `BuildingRoute` with infinite gas meter?

Reply:
Protorev's `BuildRoutes` method should not be using too much gas as all the calculation logic and tick-crossing logic is in `IterateRoutes`, are you referring to `IterateRoutes`?

I think we should add test in IterateRoutes so that we understand what our bounds are and how many ticks we want to cross while rebalancing, but I view that as a separate PR from this one (this one focused on not failing user transactions, future one focused on successfully backrunning CL pools)

Reply:
sounds good, yup i meant iterateRoutes

Reply:
changed the name: https://github.com/osmosis-labs/osmosis/pull/5841/commits/077a37195f31aa70da8096316fdc01a6b12e9366